src/vm/gccover.cpp

```diff
@@ -920,8 +921,10 @@ bool replaceInterruptibleRangesWithGcStressInstr (UINT32 startOffset, UINT32 sto
                          *((DWORD*)instrPtr) != INTERRUPT_INSTR_CALL &&
```

Comment:
I did verify that your earlier fix works for arm32 as well. For arm32 call happens via register and there is no stub involved for the jit case. So OnGcCoverageInterrupt() will be able to handle this. But I did not check for R2R case. Checking now.